ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/service/CbroTaskCreateService.java

```diff
@@ -32,11 +32,12 @@
 
   public void createTask(IGenericClient cbroClient, Task task) throws CbroTaskCreateException {
     Task t = externalizeResource(task.copy());
+    //TODO: Remove this after Logica bug response
+    t.setMeta(null);
```

Comment:
Why do we need this? the original task does not have a profile filed set anyway.

Reply:
Yes, it has. TaskBundleFactory.143 line